src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/api/ReportTests.kt

```diff
@@ -33,6 +35,27 @@ class ReportTests : IntegrationTest()
         assertSuccessfulWithResponseText(response)
         assertJsonContentType(response)
         JSONValidator.validateAgainstSchema(response.text, "Run")
+
+        val key = JSONValidator.getData(response.text)["key"].asText()
+        apiRequestHelper.delete("/reports/$key/kill/", userEmail = fakeGlobalReportReviewer())
```

Comment:
So this is a bit ugly - currently if you try to kill a report which is still queued, it fails to die and remains queued. So I'm  cleaning up this test by killing its report, so that the subsequent test for the kill endpoint can have its report run immediately and have the test succeed. We could tidy this up a bit by pulling these tests out into a separate class which always kills off any running reports between tests - but Rich has agreed that this should be fixed in orderly (this ticket: https://mrc-ide.myjetbrains.com/youtrack/issue/VIMC-4356) so that killing a queued report succeeds.